_posts/devices/2019-01-07-xdk-driver.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: page
+title:  "Xdk Driver"
+categories: [devices]
+---
+
+Eclipse Kura provides a specific driver that can be used to interact with Bosch Xdk110 devices. The driver is available only for gateways that support the new Bluetooth LE APIs. 
+It can be used in to the Wires framework, the Asset model or directly using the Driver itself.
+
+{% include alerts.html message='The Xdk driver can only be used with Xdk110 with VirtualXdkDemo installed and with firmware version> 3.5.0. If your device has an older firmware, please update it.' %}
+
+## Features
+
+The Xdk Driver can be used to get the values from all the sensor installed on the xdk110 (both in polling mode and notification):
```

Comment:
> The Xdk Driver can be used to get the values from all the sensor installed on the xdk110 (both in polling mode and notification):

The Xdk Driver can be used to get the values from all the sensors provided by the xdk110 (both in polling and notification mode):